dev/README.md

```diff
@@ -527,10 +525,80 @@ $ dsgrid query project run \
     query.json
 ```
 
+If you need to customize the Spark configuration then you will want to run the command through `spark-submit`.
+This is a bit more complicated because that tool needs to be able to locate the Python script (`dsgrid` in this case)
+and detect that it is a Python script.
+
+1. Find the location of your `dsgrid` command.
+```
+$ which dsgrid
+/Users/dthom/miniconda3/envs/dsgrid/bin/dsgrid
+```
+
+2. Substitute `dsgrid-cli.py` for the usual `dsgrid`. This allows `spark-submit` to detect that it is Python.
```

Comment:
Tell me if this workaround absolutely drives you crazy. If the average non-dsgrid-developer interacts with dsgrid through the web API, it won't be a problem.

Reply:
It depends on how big the data is. Submitting ComStock to StandardScenarios can be done on a laptop with Spark in local mode. Submitting TEMPO requires one real compute node with a Spark cluster. I'm hesitant to write a bunch of documentation with recommendations until we understand the workflow better. I am inclined to say that all jobs should be run with spark-submit because it provides the most straightforward way to customize the Spark parameters.

Consider one workflow that could be possible on either Eagle or AWS:
- We configure an always-on (or mostly-on), small virtual machine that runs a dsgrid web server.
- User sends an API command to perform action that requires a Spark cluster (data submission or query).
- dsgrid provisions the cluster based on some information provided by the user.
- The web server submits the command to the Spark cluster.